powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/database/repository/OperationRepository.java

```diff
@@ -61,4 +64,15 @@
             "ORDER BY o.timestampCreated")
     Stream<OperationEntity> findExpiredPendingOperations(Date timestamp);
 
+    @Query("SELECT o.id FROM OperationEntity o " +
+            "WHERE o.timestampExpires < :timestamp AND o.status = io.getlime.security.powerauth.app.server.database.model.OperationStatusDo.PENDING ")
+    List<String> findExpiredPendingOperationIds(Date timestamp);
+
+    @Modifying( clearAutomatically = true)
```

Comment:
The formatting is a bit off on this line (extra space). Is the `clearAutomatically` parameter required for the functionality? I scanned our whole code base and we never use this parameter.